kalliope/core/RestAPI/utils.py

```diff
@@ -32,3 +33,41 @@ def decorated(*args, **kwargs):
                 return authenticate()
         return f(*args, **kwargs)
     return decorated
+
+
+def get_parameters_from_request(http_request):
+    """
+    Get "parameters" object from the
+    :param http_request:
+    :return:
+    """
+    parameters = None
+    try:
+        # Silent True in case no parameters it does not raise an error
+        received_json = http_request.get_json(silent=True, force=True)
+        if 'parameters' in received_json:
+            parameters = received_json['parameters']
+    except TypeError:
```

Comment:
Maybe we want to `log.debug` that there is not parameters ?

Reply:
log added